src/containers/Tokens/index.js

```diff
@@ -40,8 +53,13 @@ const Tokens = props => {
       });
   }, []);
 
-  return error ? (
-    Tokens.renderError(error)
+  function renderError(errorInfo) {
+    const { result, message } = errorInfo;
+    return <NoMatch title={result} hints={[message]} />;
+  }
+
+  return error || isError ? (
```

Comment:
where is `error` defined here?

Reply:
at the top, it is being deconstructed from props